src/Allegro.DotnetSdk/Allegro.DotnetSdk.csproj

```diff
@@ -1,8 +1,8 @@
 <Project Sdk="Microsoft.Build.NoTargets">
 
   <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
-    <PackageVersion>2.1.1</PackageVersion>
+    <TargetFramework>net8.0</TargetFramework>
+    <PackageVersion>2.2.0</PackageVersion>
```

Comment:
3.0 z uwagi na 8.0?

Reply:
8.0 dla konsumentów nie ma znaczenia, nigdy nie jest sprawdzane jeśli to Sdk-reference.

Reply:
Zostawiam 2.2